arviz/tests/base_tests/test_data.py

```diff
@@ -37,6 +38,7 @@
     draws,
     eight_schools_params,
     models,
+    running_on_ci,
```

Comment:
This import is not needed anymore

Reply:
@semohr note that this is in `test_data`! not in `test_data_zarr` precisely because we ignore unused import in pylint, it does not trigger any error and we have to check manually

Reply:
Ah, I did not notice. :+1: